src/neuroconv/tools/nwb_helpers/_metadata_and_file_helpers.py

```diff
@@ -226,3 +232,75 @@ def make_or_load_nwbfile(
 
                 if not success and not file_initially_exists:
                     nwbfile_path_in.unlink()
+
+
+def resolve_backend(
```

Comment:
```suggestion
def _resolve_backend(
```

Fine to abstract this out to reduce duplication, but the phrase `resolve_backend` is a bit ambiguous

So I'd recommend marking as private (also since it's not expose to `nwb_helpers`, which is also good) just to make it clear this is an internal tool for us and our argument parsing only

Reply:
good idea